starport/chainconf/config.go

```diff
@@ -39,7 +39,7 @@ var (
 			},
 		},
 		Faucet: Faucet{
-			Port: 4500,
+			Host: "0.0.0.0:4500",
```

Comment:
```suggestion
			Host: ":4500",
```

Let's follow the same pattern here:
https://github.com/tendermint/starport/blob/e94b428a1c4856cf101463d9afc8d8d1cec632a8/starport/chainconf/config.go#L24-L30